contracts/zero-ex/contracts/src/features/TransformERC20.sol

```diff
@@ -0,0 +1,365 @@
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/v06/IERC20TokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/LibERC20TokenV06.sol";
+import "@0x/contracts-utils/contracts/src/v06/errors/LibRichErrorsV06.sol";
+import "@0x/contracts-utils/contracts/src/v06/LibSafeMathV06.sol";
+import "../errors/LibTransformERC20RichErrors.sol";
+import "../fixins/FixinCommon.sol";
+import "../migrations/LibMigrate.sol";
+import "../external/IPuppet.sol";
+import "../external/Puppet.sol";
+import "../storage/LibTransformERC20Storage.sol";
+import "../transformers/IERC20Transformer.sol";
+import "../transformers/LibERC20Transformer.sol";
+import "./ITransformERC20.sol";
+import "./ITokenSpender.sol";
+import "./IFeature.sol";
+import "./ISimpleFunctionRegistry.sol";
+
+
+/// @dev Feature to composably transform between ERC20 tokens.
+contract TransformERC20 is
+    IFeature,
+    ITransformERC20,
+    FixinCommon
+{
+    // solhint-disable const-name-snakecase
+    /// @dev Name of this feature.
+    string constant public override FEATURE_NAME = "TransformERC20";
```

Comment:
Do we have tests for accidental duplicate feature names? I'm manually checking, but I could easily see this getting overlooked.

Reply:
No, but naming and versioning features isn't essential to anything right now.